src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -100,6 +100,24 @@ private ResourceElement CreateBundle(SearchResult result, Bundle.BundleType type
                         Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(result.ContinuationToken)),
                         true);
                 }
+
+                if (result.Results.Any(x => x.IsPartial))
+                {
+                    // if the query resulted in any item indicating it is partial, add appropriate outcome
+                    // as an entry
+                    var resource = new OperationOutcome();
+                    resource.Issue = new List<OperationOutcome.IssueComponent>();
+                    resource.Issue.Add(new OperationOutcome.IssueComponent
+                    {
+                        Severity = OperationOutcome.IssueSeverity.Warning,
+                        Code = OperationOutcome.IssueType.Incomplete,
```

Comment:
I think we should provide details with text to explain what we mean by "incomplete". (`details.text` at least).